scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/tqfn.yaml

```diff
@@ -38,6 +51,64 @@ TQFN-16-1EP_3x3mm_P0.5mm_EP1.23x1.23mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+TQFN-16-1EP_5x5mm_P0.8mm_EP2.29x2.29mm:
+  device_type: 'TQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0140.PDF'
```

Comment:
Please add T1655-4

Reply:
Do you mean in the `size_source` property, e.g. like here maybe?   size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0136.PDF (T1633-5), https://pdfserv.maximintegrated.com/land_patterns/90-0032.PDF'

Or should this be in the `part_number` (I guess not, since this is essentially a generic footprint that just happens to be based on on manufacturer's datasheet)?

If so, it seems T1655-4 and T1655-4C have the same specs, should I specify both? Or just one, assuming I used one for the sizes and not the other?

Reply:
Append it to size source.
One is probably enough, otherwise the list would get quite long for some of the footprints.